packages/@expo/config/src/paths/env.ts

```diff
@@ -1,8 +1,14 @@
-import { boolish } from 'getenv';
+import { boolish, string } from 'getenv';
 
 class Env {
   /** Disable auto server root detection for Metro. This will not change the server root to the workspace root. */
   get EXPO_NO_METRO_WORKSPACE_ROOT(): boolean {
```

Comment:
The name should match the env var, e.g. `EXPO_USE_METRO_WORKSPACE_ROOT`

Reply:
We've switched the environment variable to be enabled by default: https://github.com/expo/expo/pull/30621, dropping the `EXPO_USE_METRO_WORKSPACE_ROOT`.